### DISCUSSION TITLE
How to exclude .gitignore?

### DISCUSSION THREAD

**Discussion by hf-xz** [vuejs/vitepress#3025] [Q&A] [answered]
I got:

```
The language '.gitignore' is not loaded, falling back to 'txt' for syntax highlighting.
```

while running `npm run docs:dev` or `npm run docs:build`.

The `.gitignore` is no doubt at the project root, which is also my `srcDir`.

I've tried add `'.gitignore'` or `'\.gitignore'` to my `srcExclude` but didn't work.

This message show both in dev and in build, but the path `/.gitignore` is only accessible in dev. In build-preview, the same path return a 404.

Though this has no impact to production, it's still annoying having this show up every time I start a dev server. So how could I exclude the `.gitinore`?

**Comment by brc-dd** [comment, marked as answer]
This isn't an issue with your .gitignore being included. You have some code like this somewhere in your markdown files:

````md
<<< @/.gitignore
````

or

````md
```.gitignore
...
```
````

Make a search for `.gitignore` in your project, you'll find where it's coming from.
